DependencyInjection/Compiler/AddVotersPass.php

```diff
@@ -25,13 +25,13 @@ public function process(ContainerBuilder $container)
         $voters = array();
 
         foreach ($container->findTaggedServiceIds('knp_menu.voter') as $id => $tags) {
-            foreach ($tags as $tag) {
-                $priority = isset($tag['priority']) ? (int) $tag['priority'] : 0;
-                $voters[$priority][] = $id;
+            $tag = $tags[0];
```

Comment:
are you sure about this? if its possible to add more than one of the same tag, people might have done this. should we raise an error if `count($tags) > 1`?